Framework/API/src/WorkspaceGroup.cpp

```diff
@@ -224,6 +224,38 @@ void WorkspaceGroup::print() const {
   }
 }
 
+/*
+ * @return  A non-const iterator pointing to the first workspace in this
+ *          workspace group.
+ */
```

Comment:
For doxygen comments this needs to start with `/**` not `/*` . Could you fix this in this file and the Python exports?

```
/**
 * @return  A non-const iterator pointing to the first workspace in this
 *          workspace group.
 */